virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -95,6 +95,9 @@ The SVVP Certification applies to:
 ** `KubevirtHyperconvergedClusterOperatorUSModification` has been renamed xref:../../virt/monitoring/virt-runbooks.adoc#virt-runbook-UnsupportedHCOModification[`UnsupportedHCOModification`].
 ** `SSPOperatorDown` has been renamed xref:../../virt/monitoring/virt-runbooks.adoc#virt-runbook-SSPDown[`SSPDown`].
 
+// CNV-15738: Update the documentation with supported upgrade paths
+* Retrospective upgrades between previously released z-streams are now supported.
```

Comment:
This is a bit misleading. It is giving me the impression that this is supported for `any released z-streams` based on whatever the cluster admin wants. This would not be the case though. It would however, become a possibility now, driven by what we choose to enable/disable.